models/articles.models.js

```diff
@@ -45,4 +46,33 @@ exports.selectCommentsByArticleId = (article_id) => {
     .then((result) => {
         return result.rows;
     });
+}
+
+exports.insertCommentByArticleId = (article_id, comment) => {
+
+    comment.article_id = article_id;
+    comment.votes = 0;
+    comment.created_at = Date.now();
```

Comment:
This isn't really necessary. If you have a look in the seed file you'll see that the table that has been created has this default behaviour already